pkg/descriptions.json

```diff
@@ -1997,18 +1997,11 @@
     "Name": "System.Text.Json",
     "Description": "Provides a fast, non-cached, forward-only way to read UTF-8 encoded JavaScript Object Notation (JSON) text.",
     "CommonTypes": [
+      "System.Text.Json.JsonDocument",
```

Comment:
Should these be in alphabetical order?

Reply:
mm good question, I don't think we follow patterns here as I have found both cases. Do you have a specific preference? @terrajobst @ahsonkhan

Reply:
I am fine with ordering it as what would be most commonly used.

1) JsonSerializer
2) JsonDocument
3) JsonElement
4) Utf8JsonWriter
5) Utf8JsonReader